test/tools/fuzzer_common.cpp

```diff
@@ -86,6 +91,11 @@ void FuzzerUtil::testCompiler(string const& _input, bool _optimize, bool _quiet)
 	config["settings"]["optimizer"]["enabled"] = _optimize;
 	config["settings"]["optimizer"]["runs"] = 200;
 	config["settings"]["evmVersion"] = s_evmVersions[_input.size() % s_evmVersions.size()];
+	if (_optimize && _yulOptimize)
+	{
+		config["settings"]["optimizer"]["details"] = Json::objectValue;
```

Comment:
I think you don't need this line.

Reply:
Will check it out. Actually, I realized that this fuzzer does not cover code in this directory (which I thought it would)

https://github.com/ethereum/solidity/tree/develop/libsolidity/codegen/ir

Need to check why this is so.